<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordFrame</class>
 <widget class="QWidget" name="RecordFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="childrenCollapsible">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="searchLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="UNIT_CODE">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>日期:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8" alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="pushButton">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="text">
            <string>开始查询</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/logo/pic/search.png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="6" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_04">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>客户编码:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_03">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>客户料号:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>卡板码:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>产品码:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>箱码:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_01">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>销售单号:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="CUSTOMER_ORDER_DOC_NO">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLineEdit" name="CUSTOMER_SERIAL">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="ORDER_DOC_NO">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QPushButton" name="pushButton_2">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="text">
            <string>重置查询</string>
           </property>
           <property name="icon">
            <iconset>
             <normalon>:/logo/pic/reset.png</normalon>
            </iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QDateTimeEdit" name="dateTimeEdit">
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>true</bool>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>1</second>
             <year>2000</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::DaySection</enum>
           </property>
           <property name="displayFormat">
            <string>yyyy-MM-dd</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="CUSTOMER_MATERIAL">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="BOX_CODE">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_02">
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>客户订单号:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLineEdit" name="PALLET_CODE">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PagingWidget" name="paging" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>12</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>905</width>
         <height>69</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::WrapLongRows</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="horizontalSpacing">
           <number>3</number>
          </property>
          <property name="verticalSpacing">
           <number>-1</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PagingWidget</class>
   <extends>QWidget</extends>
   <header location="global">controller.component</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>RecordFrame</receiver>
   <slot>resetEdits()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>853</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>-43</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ORDER_DOC_NO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RecordFrame</receiver>
   <slot>completerLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>-37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CUSTOMER_ORDER_DOC_NO</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RecordFrame</receiver>
   <slot>completerLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>-47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CUSTOMER_MATERIAL</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RecordFrame</receiver>
   <slot>completerLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>-87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CUSTOMER_SERIAL</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RecordFrame</receiver>
   <slot>completerLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>-48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UNIT_CODE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RecordFrame</receiver>
   <slot>completerLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>725</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordFrame</receiver>
   <slot>firstPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>711</x>
     <y>776</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BOX_CODE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RecordFrame</receiver>
   <slot>completerLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>698</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PALLET_CODE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RecordFrame</receiver>
   <slot>completerLineEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>813</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>firstPage()</slot>
  <slot>prePage()</slot>
  <slot>nextPage()</slot>
  <slot>endPage()</slot>
  <slot>jumpPage()</slot>
  <slot>dataRowSelected()</slot>
  <slot>completerLineEdit()</slot>
  <slot>resetEdits()</slot>
 </slots>
</ui>
